notes/bugfix-22443.md

```diff
@@ -0,0 +1 @@
+# # PB align widget "icon" with connectors
```

Comment:
This has an extra `#` in the beginning. Also I suggest modifying the note to:

`# Align widget "icon" with connectors in Project Browser`

Reply:
Thanks Panos, done